src/main/java/org/openjdk/jextract/JextractTool.java

```diff
@@ -364,6 +367,8 @@ private int run(String[] args) {
         parser.accepts("--output", "help.output", true);
         parser.accepts("-t", List.of("--target-package"), "help.t", true);
         parser.accepts("--version", "help.version", false);
+        parser.accepts("-F", "help.mac.framework", true);
+        parser.accepts("-framework", "help.framework.library.path", true);
```

Comment:
e.g. something simple like:
```suggestion
        if (isMacOSX) {
            parser.accepts("-F", "help.mac.framework", true);
            parser.accepts("-framework", "help.framework.library.path", true);
        }
```